tox.ini

```diff
@@ -23,29 +25,35 @@ deps =
     # install the "enum34" dependency of cryptography.
     pip>=10
 
-setenv =
-    PYTHONDONTWRITEBYTECODE = no_byte_code
-    COVERAGE_PROCESS_START = {toxinidir}/.coveragerc
-
 [testenv]
 deps =
     {[base]deps}
 extras = all, test
 
-whitelist_externals =
-    sh
-
 setenv =
-    {[base]setenv}
+    # use a postgres db for tox environments with "-postgres" in the name
+    # (see https://tox.readthedocs.io/en/3.20.1/config.html#factors-and-factor-conditional-settings)
     postgres: SYNAPSE_POSTGRES = 1
+
+    # this is used by .coveragerc to refer to the top of our tree.
     TOP={toxinidir}
 
 passenv = *
 
 commands =
-    /usr/bin/find "{toxinidir}" -name '*.pyc' -delete
```

Comment:
> bytecode files aren't called `*.pyc` under python 3.

Aren't they? I have plenty of `*.pyc` files in my local working directory, albeit inside `__pycache__/` folders instead of directly alongside the source Python files like in older Python versions.

That said, it's not obvious to me why we're deleting them (or preventing their generation) in the first place.

Reply:
oh, sorry, that's right. They're just in a different folder; they still have the .pyc name.

I still think we should remove the removal.